frontend/src/views/web/update-profile.html

```diff
@@ -32,6 +32,15 @@
                             <p ng-message="maxlength"> Last Name is too long. </p>
                         </div>
                     </div>
+                    <div class="input-field align-left"> 
+                        <input id="affiliation" name="affiliation" type="text" class="text-dark-black dark-autofill w-400" ng-model="updateProfile.user.affiliation" ng-maxlength="30" ng-pattern="/^[A-Za-z]*$/" required focus-if="updateProfile.user.affiliation">
```

Comment:
@anirudhvenu I think there is no need for `ng-pattern` in affiliation. Is there any specific reason for using it that I am missing ?